CHANGELOG.rst

```diff
@@ -24,7 +24,7 @@ Backward-incompatible changes:
 Changes:
 ^^^^^^^^
 
-*none*
+- `pem.twisted.certificateOptionsFromPEMs` will now load Ephemeral Diffie-Hellman parameters if found.
```

Comment:
This is reST, so you have to use double backticks. :(

Also please add a link to the PR in a second line like in the other entries that have one.